src/main/java/com/google/sps/data/LearningPathService.java

```diff
@@ -136,16 +152,73 @@ public LearningItem loadItem(long itemId) throws EntityNotFoundException {
 		return mapEntityToLearningItem(itemId, item);
 	}
 
-	private LearningItem mapEntityToLearningItem(long sectionId, Entity e) {
+    public LearningItem loadFeedbackItems(long itemId) throws EntityNotFoundException {
+		Entity item = datastore.get(KeyFactory.createKey(LEARNING_ITEM, itemId));
+		return mapEntityToLearningItem(itemId, item);
+	}
```

Comment:
Generally I think we only need to load feedback by (user+learning item). 
 - I think we can get away without a 'load by ID' for this entity
 - the function name (plural) doesn't match the return type (singular).